client/landing/gutenboarding/onboarding-block/domains/index.tsx

```diff
@@ -80,13 +80,21 @@ const DomainsStep: React.FunctionComponent< Props > = ( { isModal } ) => {
 		</div>
 	);
 
+	const trackDomainSearchInteraction = ( query: string ) => {
+		trackEventWithFlow( 'calypso_newsite_domain_search_blur', {
+			query,
+			where: isModal ? 'domain_modal' : 'domain_page',
```

Comment:
@alshakero @yansern @razvanpapadopol FYI, some of these events could do with "flow: onboarding|launch` treatment.